waving_hands/gamemaster.py

```diff
@@ -3,19 +3,41 @@
 import select
 import socket
 import sys
+import logging
+import logging.config
+
+from waving_hands.elemental import Elemental
+from waving_hands.minion import Minion
+from waving_hands.targetable_client import TargetableClient
+from waving_hands.wizard import Wizard
+from waving_hands import groblenames
+
+log = logging.getLogger(__name__)
```

Comment:
Logs here typically all get their own modules. Logging produces output based on the formatters, and looks something like this: 

```
[INFO] waving_hands.gamemaster::wait_for_connections() Listening for 2 players on ('localhost', 12345)
```
Which is handy for finding the function where things are being called. There are a lot of other handy things in formatters: 
https://docs.python.org/3/library/logging.html#logrecord-objects